File: VB/E911.Module.Win/WinModule.vb
Imports System.ComponentModel
Imports DevExpress.ExpressApp

Namespace E911.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class E911WindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Public Overrides Sub ExtendModelInterfaces(ByVal extenders As DevExpress.ExpressApp.Model.ModelInterfaceExtenders)
			MyBase.ExtendModelInterfaces(extenders)
			extenders.Add(Of E911.Module.Editors.IModelCustomUserControlViewItem, E911.Module.Win.Editors.IModelWinCustomUserControlViewItem)()
		End Sub
	End Class
End Namespace

File: CS/E911.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using DevExpress.ExpressApp;

namespace E911.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class E911WindowsFormsModule : ModuleBase {
        public E911WindowsFormsModule() {
            InitializeComponent();
        }
        public override void ExtendModelInterfaces(DevExpress.ExpressApp.Model.ModelInterfaceExtenders extenders) {
            base.ExtendModelInterfaces(extenders);
            extenders.Add<E911.Module.Editors.IModelCustomUserControlViewItem, E911.Module.Win.Editors.IModelWinCustomUserControlViewItem>();
        }
    }
}
